panda/src/bullet/bulletWheel.cxx

```diff
@@ -202,7 +202,7 @@ get_wheel_radius() const {
 void BulletWheel::
 set_steering(PN_stdfloat value) {
 
-  _info.m_steering = (btScalar)value;
+  _info.m_steering = (btScalar)deg_2_rad(value);
```

Comment:
This seems like a functional change, one that also breaks compatibility.  Could you explain it?